### PR TITLE
Markdown fixer: fix front matter bug with no space

### PR DESCRIPTION
**Pull request by araslanov-e**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update

## Description
Regexp `\s?` for some reason did not take into account the absence of a space

## Related Tickets & Documents
https://github.com/thepracticaldev/dev.to/issues/3846

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

**Comment by rhymes** [comment]
Hi @araslanov-e, thanks for so many PRs! I think this is based on #3846, could you base it on master? Because it contains the changes of another PR, or if they are tied maybe we could make it clear that this depends on the other one

**Comment by araslanov-e** [comment]
@rhymes Ooops. Done!

File: app/labor/markdown_fixer.rb [hunk 1]
```diff
@@ -54,10 +54,10 @@ def underscores_in_usernames(markdown)
 
     def add_quotes_to_section(markdown, section:)
       # Only add quotes to front matter, or text between triple dashes
-      markdown.gsub(FRONT_MATTER_DETECTOR) do |front_matter|
-        front_matter.gsub(/#{section}:\s?(.*?)(\r\n|\n)/m) do |target|
-          # 1 is the captured group (.*?)
-          captured_text = Regexp.last_match(1)
+      markdown.sub(FRONT_MATTER_DETECTOR) do |front_matter|
```
Comment (maestromac) [inline app/labor/markdown_fixer.rb]: Changing `gsub` to `sub` is a nice touch.

Comment (araslanov-e) [reply app/labor/markdown_fixer.rb]: B)

**Pull request merged**